src/components/gridList/useGridLayout.ts

```diff
@@ -42,18 +43,22 @@ const useGridLayout = (props: GridListBaseProps) => {
   }, [numberOfColumns, itemSpacing, keepItemSize ? undefined : _containerWidth]);
 
   const itemContainerStyle = useMemo(() => {
-    return {width: itemWidth /* + itemSpacing */, marginRight: itemSpacing, marginBottom: itemSpacing};
+    return {width: itemWidth /* , marginRight: itemSpacing, marginBottom: itemSpacing */};
   }, [itemWidth, itemSpacing]);
 
   const listContentStyle = useMemo(() => {
-    return [{paddingHorizontal: listPadding}, contentContainerStyle];
-  }, [listPadding, contentContainerStyle]);
-  
+    return [{/* paddingHorizontal: listPadding, */ rowGap: itemSpacing}, contentContainerStyle];
+  }, [listPadding, itemSpacing, contentContainerStyle]);
```

Comment:
We have extra dependencies now (here and above) from the commented code

Reply:
Fixed